jobs/ci-kubernetes-e2e-gce-sd-logging.env

```diff
@@ -0,0 +1,9 @@
+### job-env
+PROJECT=k8s-jkns-gce-sd-log
```

Comment:
There's an internal script we use to ensure this file has the right grants. Did you use this?